pkg/resume.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright AppsCode Inc. and Contributors
+
+Licensed under the AppsCode Community License 1.0.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://github.com/appscode/licenses/raw/1.0.0/AppsCode-Community-1.0.0.md
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pkg
+
+import (
+	cs "stash.appscode.dev/apimachinery/client/clientset/versioned"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func NewCmdResume(clientGetter genericclioptions.RESTClientGetter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "resume",
+		Short:             `resume stash resources`,
+		DisableAutoGenTag: true,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+
+			cfg, err := clientGetter.ToRESTConfig()
+			if err != nil {
+				return errors.Wrap(err, "failed to read kubeconfig")
+			}
+
+			namespace, _, err = clientGetter.ToRawKubeConfigLoader().Namespace()
+			if err != nil {
+				return err
+			}
+
+			stashClient, err = cs.NewForConfig(cfg)
+			if err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+	cmd.AddCommand(NewCmdResumeBackup())
+	cmd.PersistentFlags().StringVar(&backupConfig, "backup-config", backupConfig, "Name of the Backupconfiguration")
+	cmd.PersistentFlags().StringVar(&backupBatch, "backup-batch", backupBatch, "Name of the BackupBatch")
```

Comment:
Rename `backup-config` and `backup-batch` to `backupconfig` and `backupbatch` respectively.